CHANGELOG.md

```diff
@@ -3,6 +3,18 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.1dev
```

Comment:
Don't forget to bump the version to v2.0.0

Reply:
done